apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameter.cs

```diff
@@ -137,11 +104,56 @@ public override ParameterDirection Direction
 #endif
 
         /// <summary>
-        /// The <see cref="SpannerDbType"/> of the parameter or column. This should match the type as defined in Spanner
+        /// The <see cref="SpannerDbType" /> of the parameter or column. This should match the type as defined in Spanner
         /// or as defined by the result of a SQL Query.
         /// </summary>
         public SpannerDbType SpannerDbType { get; set; }
 
+        /// <inheritdoc />
+        public override object Value { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the SpannerParameter class.
+        /// </summary>
+        public SpannerParameter() { }
+
+        /// <summary>
+        /// Initializes a new instance of the SpannerParameter class.
+        /// </summary>
+        /// <param name="parameterName">
+        /// The name of the parameter. For Insert, Update and Delete commands, this name should
+        /// be the name of a valid column in a Spanner table. In Select commands, this name should be the name of a parameter
+        /// used in the SQL Query. This value is case sensitive. Must not be null.
+        /// </param>
+        /// <param name="type">
+        /// One of the <see cref="SpannerDbType" /> values that indicates the type of the parameter.
+        /// Must not be null.
+        /// </param>
+        /// <param name="value">An object that is the value of the SpannerParameter. May be null.</param>
+        /// <param name="sourceColumn">
+        /// The name of the DataTable source column (SourceColumn) if this SpannerParameter is
+        /// used in a call to Update. May be null.
+        /// </param>
+        /// <param name="size">The length of the parameter. The value is for informational purposes only.</param>
```

Comment:
made more explicit.  its not used to enforce anything, though it might be used in some datagridview scenarios to enforce some behavior in the UI.  Our library does nothing with size however.